### ISSUE TITLE
Enable manual import path specification

### ISSUE THREAD

**Issue by mathsaey** [ahamez/protox#4]
Providing multiple files to the `files` directive will automatically use the common_path (provided by the `common_directory_path` function) as `protoc`'s import directory.

For instance, if you have the following directory layout:

```
some/prefix/path/foo.proto
some/prefix/path/bar/baz.proto
```

protox will automatically use `some/prefix/foo` as the import path.

This can be problematic if `baz.proto` uses `some/prefix/path` in its import statement:

```
syntax = "proto3";

import "some/prefix/path/foo.proto";

...
```

The current path handling will return a `some/prefix/path/foo.proto: File not found.` error.

Would it be possible to add a `path` argument to the `use` statement which makes it possible to work around this behavior?

**Comment by mathsaey** [comment]
FWIW: I forked the repo and implemented this feature. Once I have some time to write documentation and tests I'll be sure to send a pull request.

**Comment by ahamez** [comment]
Closed by #5.
